frontend/src/styles/notification.css

```diff
@@ -0,0 +1,8 @@
+.person-name {
+  color: #4FAF4E; }
+
+.time {
+  color: #484848; }
+
+.notif-description {
+  color: #484848; }
```

Comment:
I would move these variables into the variables.scss files for more modularity. Also make sure that only the scss files are being updated since they will be compiled into the css pages.